ra/ra.go

```diff
@@ -264,19 +264,45 @@ func validateEmail(ctx context.Context, address string, resolver bdns.DNSClient)
 	return emptyDNSResponseError
 }
 
+// certificateRequestAuthz is a struct for holding information about a valid
+// referenced during a certificateRequestEvent. It holds both the authorization
+// ID and the challenge type that made the authorization valid.
+type certificateRequestAuthz struct {
+	ID            string `json:",omitempty"`
```

Comment:
Given neither of these should never be default values I think it's safe to just remove the `json` field tags entirely.

Reply:
Addressed with e03f6ab